### ISSUE TITLE
[1.11.2 v0.014] BloodMagic-1.11-2.1.7-76 compatibility issue?

### ISSUE THREAD

**Issue by WhiskersRS** [RootsTeam/Roots#205]
When attempting to use the Inspectoris Scandalum (from Blood Magic) to see a preview of the blood alter upgrades no preview appears with any tier. After attempting to use the book the spells Gravity Boost and Growth Infusion no longer work. With Gravity Boost the spell effects occur and the correct petal dust is used up but I move nowhere.

The spells Shatter, Petal Shell and Light Drifter work correctly, I have not tried the others.

Exiting and restarting Minecraft allows the spells to work again, until the book is used. The book is always unusable, I haven't tested it without Roots 2.

Using Forge 1.11.2-13.20.0.2229

**Comment by elucent** [comment]
That is...strange. I don't suppose there's any log information?

**Comment by WhiskersRS** [comment]
Please forgive my ignorance I am new to this, what do you need exactly?
I am guessing the logs from .minecraft/logs folder?
Should I post the entire log?

**Comment by elucent** [comment]
Yes, in a site like Gist or Pastebin or something. Just in case there's anything there, since I basically have nothing else to go on.

**Comment by WhiskersRS** [comment]
3 days of logs
http://pastebin.com/d7zbs67e
http://pastebin.com/wbymzDSX
http://pastebin.com/24K9fsYJ
